Cannot build Gradle Android with current Android SDK

**Describe the bug**
After starting a new project, and installing the latest Android SDK, it is impossible to build the Gradle Android project on Windows.

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new Codename One Maven project for IntelliJ
2. Run the Local Builds / Gradle Android Project run configuration
3. Observe the following error chain in the logs:
```
[ERROR] Failed to execute goal com.codenameone:codenameone-maven-plugin:7.0.58:build (build-android) on project XXX-android: Failed to build android app: Failed to get SDK list using XXX\Android\sdk\tools\bin\sdkmanager.bat: Cannot run program "XXX\Android\sdk\tools\bin\sdkmanager.bat": CreateProcess error=2, The system cannot find the file specified -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.codenameone:codenameone-maven-plugin:7.0.58:build (build-android) on project geekzonemobile-android: Failed to build android app
...
Caused by: org.apache.maven.plugin.MojoExecutionException: Failed to build android app
...
Caused by: com.codename1.builders.BuildException: Failed to get SDK list using XXX\Android\sdk\tools\bin\sdkmanager.bat    
...
Caused by: java.io.IOException: Cannot run program "XXX\Android\sdk\tools\bin\sdkmanager.bat": CreateProcess error=2, The system cannot find the file specified
...
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
```

**Expected behavior**
The Gradle Android project should be created

**Additional context**
According to [#2816](https://github.com/codenameone/CodenameOne/issues/2816#issuecomment-496186168), the root cause of this issue is that newer versions of Android Studio have changed the paths.  

However, the recommended solution of downloading the older version of the SDK is not a great one, as there does not appear to be an official (and therefore safe!) source for these tools.

Instead, the CN1 Plugin should consider both the legacy and current tool paths.

--- a/maven/codenameone-maven-plugin/src/main/java/com/codename1/builders/AndroidGradleBuilder.java
+++ b/maven/codenameone-maven-plugin/src/main/java/com/codename1/builders/AndroidGradleBuilder.java
@@ -395,8 +395,14 @@ public class AndroidGradleBuilder extends Executor {
         }
 
         File sdkmanager = new File(androidSDKDir, path("tools", "bin", "sdkmanager"+bat));
-
-
+        if (!sdkmanager.canExecute()) {
+            sdkmanager = new File(androidSDKDir, path("cmdline-tools", "latest", "bin", "sdkmanager"+bat));
+        }
+        if (!sdkmanager.canExecute()) {
+            Exception ex = new RuntimeException("Android SDK Command-Line Tools not found");
+            error("Cannot find executable sdkmanager"+bat+" in "+androidSDKDir+"; tried tools and cmdline-tools/latest", ex);
+            throw new BuildException("Cannot find executable sdkmanager"+bat+" in "+androidSDKDir+"; tried tools and cmdline-tools/latest", ex);
+        }
 
         String sdkListStr;
         try {

